packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -1806,9 +1806,15 @@ export const audiusBackend = ({
   }
 
   // Favoriting a track
-  async function saveTrack(trackId: ID) {
+  async function saveTrack(
+    trackId: ID,
+    metadata?: { is_save_of_repost: boolean }
+  ) {
     try {
-      return await audiusLibs.EntityManager.saveTrack(trackId)
+      return await audiusLibs.EntityManager.saveTrack(
+        trackId,
+        JSON.stringify(metadata)
```

Comment:
just curious if we need to json.stringify here. interesting that function doesn't do it for us? i guess would require libs update otherwise?

Reply:
yea that function requires a string, and all social functions in entity manager libs have the same arguments, so expecting string inputs was cleaner than expecting diff dicts